HW3/BigestWordSearch.py

```diff
@@ -0,0 +1,11 @@
+data = str(input('Enter some text: '))
+chars = ". , ; : ' ! & $ % ( ) + - * / =".split()
```

Comment:
if I would write brackets w/o parentheses it would be wrong answer
i.e. `word(other) next` will transform to `wordother`